sdk/sdk-api/src/main/java/mil/emp3/api/utils/kml/KMLExportThread.java

```diff
@@ -530,6 +545,46 @@ public void serializeGeometry(final XmlSerializer xmlSerializer) throws IOExcept
         return oArray;
     }
 
+    private static String convertDocumentToString(final Document doc) {
+        final TransformerFactory tf = TransformerFactory.newInstance();
+        try {
+            final Transformer transformer = tf.newTransformer();
+            // below code to remove XML declaration
+            transformer.setOutputProperty(OutputKeys.OMIT_XML_DECLARATION, "yes");
+            final StringWriter writer = new StringWriter();
```

Comment:
This is a resource that needs to be closed.  Please wrap this around a try with resources block so that it will close the resource without having a memory leak. if you need help let me know!